examples/Test_domain/connectivity/2000/example_conn.csv

```diff
@@ -1,6 +1,6 @@
 # Transfer probability matrix for tests only,
```

Comment:
You missed one

Reply:
All fixed now, I thought it was part of the sentence

Reply:
Tests all passing